src/seedu/addressbook/storage/jaxb/AdaptedPerson.java

```diff
@@ -31,6 +32,8 @@
     private AdaptedContactDetail email;
     @XmlElement(required = true)
     private AdaptedContactDetail address;
+    @XmlElement
```

Comment:
If you trying work incrementally, don't do load/saving first. Check if you feature works locally